recipes/mfast/all/conanfile.py

```diff
@@ -71,10 +71,12 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("boost/1.75.0")
+        # mfast<=1.22.0 doesn't support boost>=1.76.0
+        # see https://github.com/objectcomputing/mFAST/issues/118
+        self.requires("boost/1.75.0", transitive_headers=True)
         self.requires("tinyxml2/9.0.0")
```

Comment:
Seems like the error comes from here https://c3i.jfrog.io/c3i/misc-v2/logs/pr/19697/2-macos-clang/mfast/1.2.2//34ec7872f5b8410cae94a29f180bdef476275d36-test.txt

Maybe a missing trait here?

Reply:
More likely SIP.